### DISCUSSION TITLE
Multichannel and Scan Mode Simultaneously

### DISCUSSION THREAD

**Discussion by tet024** [rtl-airband/RTLSDR-Airband#280] [Q&A]
Hello, 

So I have 2 usb dongles and I would like to have one setup in scan mode and the other in multichannel mode is this possible ?

**Comment by charlie-foxtrot** [comment]
yes, `mode` is set per-device, see https://github.com/szpajder/RTLSDR-Airband/wiki/Configuring-RTLSDR-devices (or similar section based on device type)
